UI/Marker/EnemyPlacemark.gd

```diff
@@ -8,10 +8,12 @@ const COLOR_DEFAULT = Color.white
 export (Resource) var enemy: Resource setget set_enemy
 
 onready var _detailsPanel: DetailsPanel = DetailsPanel.get_instance(get_tree())
+onready var is_selected = false
+var list_manager = SelectedListManager
```

Comment:
Instead of storing it in a variable directly just use SelectedListManager when you need